entrypoint.sh

```diff
@@ -79,6 +79,13 @@ if [ "$1" = "kubelet" ]; then
 
     # separate flow for cri-dockerd to minimize change to the existing way we run kubelet
     if [ "${RKE_KUBELET_CRIDOCKERD}" == "true" ]; then
+
+        # Mount kubelet docker config to /.docker/config.json
+        if [ ! -z "${RKE_KUBELET_DOCKER_CONFIG}" ]
+        then
+          mkdir -p /.docker && touch /.docker/config.json
+          mount --bind ${RKE_KUBELET_DOCKER_FILE} /.docker/config.json
```

Comment:
prefixPath is already present in the env `RKE_KUBELET_DOCKER_FILE` so not adding it here